src/pages/login/login.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {
+    useHistory,
+} from 'react-router-dom';
+
+function Login() {
+    const history = useHistory();
+    function HandleClick() {
```

Comment:
de HandleClick function moet niet met een hoofdletter, en in deze functie moet de toggleIsAuthenticated naar true gezet worden door dit binnen de acolades van de functie te zetten:

toggleIsAuthenticated(true)